pkg/permissions/generate.go

```diff
@@ -19,6 +20,80 @@ type AgentRBACManifests struct {
 	RoleBindings []rbac.RoleBinding
 }
 
+//*******************************
+func AgentCLR(clusterRole []rbac.ClusterRole) string {
+	var got []string
+	for _, clr := range clusterRole {
+		var aG, rC, vB string
+		for _, rule := range clr.Rules {
+			aG = strings.Join(rule.APIGroups, "\", \"")
+			rC = strings.Join(rule.Resources, "\", \"")
+			vB = strings.Join(rule.Verbs, "\", \"")
+		}
+
+		got = append(got, fmt.Sprintf(
+			`apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+	name: jetstack-secure-agent-pods-reader
+rules:
+- apiGroups: ["%s"]
+	resources: ["%s"]
+	verbs: ["%s"]`, aG, rC, vB))
+	}
+	out := strings.Join(got, "\n")
+	return out
+}
+
+//*******************************
+func AgentCLRB(CLRB []rbac.ClusterRoleBinding) string {
```

Comment:
```suggestion
func agentCLRB(clrbs []rbac.ClusterRoleBinding) string {
```

Reply:
could we name the function something more clear, expressing what the function does?